tests/unit/amo/reducers/test_versions.js

```diff
@@ -36,7 +37,7 @@ describe(__filename, () => {
       fetchVersions({ errorHandlerId: 1, slug }),
     );
 
-    expect(getVersionsBySlug({ slug, state })).toBe(null);
+    expect(getVersionsBySlug({ slug, state })).toEqual([]);
```

Comment:
Similar to my comment on the last review, we usually set data to null when fetching. Otherwise, how would you be able to know if the API returned a legitimately empty array of versions?

Reply:
I've changed that back.